### PR TITLE
[Statusbar] Add media controller panel

### PR DESCRIPTION
**Pull request by neochapay**

### CODE REVIEW

File: src/qml/Statusbar.qml [hunk 1]
```diff
@@ -289,9 +291,18 @@ Item {
             iconSize: Theme.itemHeightExtraSmall
             source: "image://theme/icon_gps_normal"
         }
+
         StatusbarItem {
             iconSize: Theme.itemHeightExtraSmall
-            source: "image://theme/icon_play_pause"
+            source: (mprisManager.currentService && mprisManager.playbackStatus == Mpris.Playing) ?
+                        "/usr/share/themes/glacier/fontawesome/icons/pause.png"
```
[the remark this answers is not in the document]
Comment (neochapay) [reply src/qml/Statusbar.qml]: Hulk to break

[the remark this answers is not in the document]
Comment (neochapay) [reply src/qml/Statusbar.qml]: ready

File: src/qml/Statusbar.qml [hunk 2]
```diff
@@ -289,9 +291,18 @@ Item {
             iconSize: Theme.itemHeightExtraSmall
             source: "image://theme/icon_gps_normal"
         }
+
         StatusbarItem {
             iconSize: Theme.itemHeightExtraSmall
-            source: "image://theme/icon_play_pause"
+            source: (mprisManager.currentService && mprisManager.playbackStatus == Mpris.Playing) ?
+                        "image://theme/pause"
```
Comment (eekkelund) [inline src/qml/Statusbar.qml]: I haven't got pause button visible even though song is paused.

File: src/qml/statusbar/MediaController.qml [hunk 3]
```diff
@@ -0,0 +1,139 @@
...
+                          var artistTag = Mpris.metadataToString(Mpris.Artist)
+                          var titleTag = Mpris.metadataToString(Mpris.Title)
+
+                          var artistLabel = (artistTag in mprisManager.metadata) ? mprisManager.metadata[artistTag].toString() : "";
+                          var titleLabel = (titleTag in mprisManager.metadata) ? mprisManager.metadata[titleTag].toString() : ""
+
+                          return artistLabel+" - "+titleLabel
+
+                      }else{
+                          return qsTr("Player not start")
```
Comment (eekkelund) [inline src/qml/statusbar/MediaController.qml]: Maybe better would be "No music playing" or "Music player not active" or "Music player haven't been started".
I think myself that the best would be "No music playing"

File: src/qml/statusbar/MediaController.qml [hunk 4]
```diff
@@ -0,0 +1,139 @@
...
+                id: trackName
+                width: parent.width
+                text: if (mprisManager.currentService) {
+                          var artistTag = Mpris.metadataToString(Mpris.Artist)
+                          var titleTag = Mpris.metadataToString(Mpris.Title)
+
+                          var artistLabel = (artistTag in mprisManager.metadata) ? mprisManager.metadata[artistTag].toString() : "";
+                          var titleLabel = (titleTag in mprisManager.metadata) ? mprisManager.metadata[titleTag].toString() : ""
+
+                          return artistLabel+" - "+titleLabel
```
Comment (eekkelund) [inline src/qml/statusbar/MediaController.qml]: Maybe in two rows? Then also the long song names or artist names are fully visible.
eg.
"Put Your Hands Up For Detroit - Fedde Le Grand"
vs
"Fedde Le Grand
Put Your Hands Up For Detroit"

**Comment by eekkelund** [comment]
@neochapay How is this going?:) If I remember right you had fixes ready?

**Comment by neochapay** [comment]
@eekkelund oh sorry

**Comment by eekkelund** [comment]
LGTM

**Pull request merged**